tests/foreman/ui/test_contenthost.py

```diff
@@ -315,6 +315,46 @@ def test_positive_search_by_subscription_status(session, vm):
         assert values['table'][0]['Name'] == vm.hostname
 
 
+@pytest.mark.tier3
+def test_positive_toggle_subscription_status(session, vm):
+    """Register host into the system, assert subscription status valid,
+    toggle status off and on again using CLI and assert status is updated in web UI.
+
+    :id: b9343e6f-9354-46ef-873f-b63851d29043
+
+    :expectedresults: Subscription status changed on the CLI is reflected in the web UI.
+
+    :customerscenario: true
+
+    :BZ: 1836868
+
+    :CaseLevel: System
+
+    :CaseImportance: Medium
+    """
+    with session:
+        session.location.select('Default Location')
+        assert session.contenthost.search(vm.hostname)[0]['Name'] == vm.hostname
+        subscriptions = session.contenthost.read(vm.hostname, widget_names='subscriptions')[
+            'subscriptions'
+        ]
+        assert subscriptions['auto_attach'] == 'Yes'
```

Comment:
```suggestion
        assert subscriptions['auto_attach'].lower() == 'yes'
```

Non-blocking comment! Making the step future proof.

Reply:
OK